packages/nextjs/app/page.tsx

```diff
@@ -54,10 +53,10 @@ const Home: NextPage = () => {
                 height={25}
               ></Image>
               <p>
-                Tinker with your smart contract using the{" "}
+                Tinker with your smart contract using the
```

Comment:
Be careful with this change, check

![Captura desde 2024-08-29 09-44-33](https://github.com/user-attachments/assets/be84843f-4299-4025-b07c-d3f314a9623e)
![Captura desde 2024-08-29 09-44-25](https://github.com/user-attachments/assets/d15a4d18-38ef-4d8b-b714-f406c0626bde)

Reply:
Sorry for the missing check